modules/claims_api/lib/fes_service/README.md

```diff
@@ -0,0 +1,137 @@
+# FES Service API Documentation
```

Comment:
We generally don't add README files as part of service implementations. Might be good to save this & later add to documentation elsewhere though.